api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
Now I remember why we had other statuses such as `Paused`, `Continuing`, and `Stepping`. Once you hit a `Paused` status, to signal that you want to step, you request `Stepping` which will result in a step and return of `Paused`. Just a `Running` boolean isn't enough to do anything but continue to the next breakpoint.